### DISCUSSION TITLE
What happens to branches after merging in a pull request?

### DISCUSSION THREAD

**Discussion by mekdimd** [ssss-sfu/ssss-sfu.github.io#79] [Q&A] [answered]
What should be done after merging a branch, should we delete it or keep it hanging around? This is not specified in [CONTRIBUTING.MD](https://github.com/ssss-sfu/ssss-sfu.github.io/blob/master/CONTRIBUTING.md)

**Comment by brianrahadi** [comment, marked as answer]
Hey Mekdim! Thanks for asking!
We haven't decided on this, but it's best to delete it if it's merged main. 
If it's merged to other branches, it's okay to still hang it and delete it if the other branches are merged to the main.

I will work on cleaning the stale branches.
